lib/awful/layout/suit/spiral.lua

```diff
@@ -25,13 +26,22 @@ local math = math
 local spiral = {}
 
 local function do_spiral(p, is_spiral)
+    local t = p.tag or capi.screen[p.screen].selected_tag
     local wa = p.workarea
     local cls = p.clients
     local n = #cls
     local old_width, old_height = wa.width, 2 * wa.height
+    local mwfact = t.master_width_factor + 0.5
```

Comment:
Why `+ 0.5`?

Reply:
This is nessecary for it the default to behave as it did before this patch. Without changing the mwfact by pressing the key, the windows are split in the center of the screen. Without adding `+ 0.5`, the default center would have the left window occupy only a small part on the left of the screen and it cannot be resized until the right side.